src/components/CheckboxCollapse.tsx

```diff
@@ -0,0 +1,112 @@
+import { Checkbox, CheckboxChangeEvent } from "antd";
```

Comment:
The new component!

![Image](https://github.com/user-attachments/assets/fc6a59fa-cf2e-4900-8e18-96d20be94f48)